src/module/enforce.wren

```diff
@@ -0,0 +1,43 @@
+class Error {
+  toString { message }
+  raise() { Fiber.abort(this) }
+}
+
+class ArgumentError is Error {
+  construct new(message) {
+    _message = message
+  }
+  message { _message }
+}
```

Comment:
This kind of exception hierarchy is clearly something we want in Wren. It could be moved to its own module.
```js
import "errors" for ArgumentError
```
Which will make it easy to extend
``````js
import "errors" for Error
class MyError {
  construct new() {}
  message { "My errors have a constant error message." }
}
```

Reply:
That's where I'm headed, one thing at a time. :-)